refl1d/polymer.py

```diff
@@ -1089,4 +1095,4 @@ def _calc_g_zs_uniform(self, g_z, g_zs):
             segments = g_zs.shape[1]
             pg_zs = g_zs[:, 0]
             for r in range(1, segments):
-                g_zs[:, r] = pg_zs = correlate(pg_zs, LAMBDA_ARRAY, 1) * g_z
+                g_zs[:, r] = pg_zs = correlate(pg_zs, LAMBDA_ARRAY, 1) * g
```

Comment:
I messed up when merging from master.  This should be `... * g_z`.  Other than that, these are only linting changes